### PR TITLE
feat(run benchy remotely)

### PR DESCRIPTION
**Pull request by laser**
Makes incremental progress towards #761 

## What's in this PR?

This changeset includes a script which will run `benchy.sh` on a remote server via SSH and write its output to the client's stdout.

### CODE REVIEW

File: fil-proofs-tooling/scripts/benchy-remote.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMDS=$(cat <<EOF
+cd \$(mktemp -d)
+git clone -q https://github.com/filecoin-project/rust-fil-proofs.git
+cd rust-fil-proofs
+git checkout -q master
+/root/.cargo/bin/cargo build --release --all > /dev/null 2>&1
```
Comment (dignifiedquire) [inline fil-proofs-tooling/scripts/benchy-remote.sh]: we should always build with `RUSTFLAGS="-C target-cpu=native"` for benchmarks

File: fil-proofs-tooling/scripts/benchy-remote.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMDS=$(cat <<EOF
+cd \$(mktemp -d)
+git clone -q https://github.com/filecoin-project/rust-fil-proofs.git
+cd rust-fil-proofs
+git checkout -q master
```
Comment (dignifiedquire) [inline fil-proofs-tooling/scripts/benchy-remote.sh]: what does `-q` do?

Comment (laser) [reply fil-proofs-tooling/scripts/benchy-remote.sh]: It puts git in quiet mode, i.e. suppresses confirmation output

File: fil-proofs-tooling/scripts/benchy-remote.sh (hunk 1 above)
Comment (laser) [reply fil-proofs-tooling/scripts/benchy-remote.sh]: Done!

**Pull request merged**